src/ua-parser.js

```diff
@@ -270,6 +270,12 @@
             /(yabrowser)\/([\w\.]+)/i                                           // Yandex
             ], [[NAME, 'Yandex'], VERSION], [
 
+            /(Avast)\/([\w\.]+)/i                                               // Avast Secure Browser
+            ], [[NAME, 'Avast Secure Browser'], VERSION], [
+
+            /(AVG)\/([\w\.]+)/i                                                 // Avast Secure Browser
```

Comment:
The comment here should be `AVG Secure Browser`.

Reply:
fixed